mentoring-app/src/app/courses-page/courses-page.module.ts

```diff
@@ -45,6 +46,7 @@ const courseRoutes: Routes = [
     FormsModule,
     MatButtonModule,
     MatInputModule
-  ]
+  ],
+  providers: [CoursesService]
```

Comment:
Why do you prefer injecting your services into @NgModule to injecting it in the service class itself? What about Tree-Shakable providers?

Reply:
fixed